internal/xds/translator/testdata/out/xds-ir/accesslog-als-tcp.clusters.yaml

```diff
@@ -41,13 +41,6 @@
         typedConfig:
           '@type': type.googleapis.com/envoy.extensions.transport_sockets.raw_buffer.v3.RawBuffer
   type: EDS
-  typedExtensionProtocolOptions:
```

Comment:
why are these getting removed ?

Reply:
because it's marked as using proxy protocol.